node-tests/acceptance/exam-test.js

```diff
@@ -327,6 +339,7 @@ describe('Acceptance | Exam Command', function() {
       ]).then(child => {
         const output = child.stdout;
         assertTestExecutionJson(output);
+        assertTestExecutionFailedBrowsers(output, 0);
```

Comment:
it seems like we are getting testExecution path twice, one from `assertTestExecutionJson` and `assertTestExecutionFailedBrowsers`. Can we have function to get the execution file path/json file and pass the path to functions so that we don't read json file twice?

Reply:
combined the two assert functions!